build.sh

```diff
@@ -91,14 +91,14 @@ if [ "$LINK_TYPE" == "shared" ]; then
 fi
 
 # Set target MacOS minver
-export MACOSX_DEPLOYMENT_TARGET=10.10
+export MACOSX_DEPLOYMENT_TARGET=11.10
```

Comment:
I would like others to chime in if it is OK to upgrade that one. I do realize that for arm64 silicon you need the new tools / SDK.

Reply:
I made adjustments: 11.1 just for arm64